src/libYARP_dev/src/modules/ControlBoardWrapper/RPCMessagesParser.cpp

```diff
@@ -2081,6 +2089,22 @@ bool RPCMessagesParser::respond(const yarp::os::Bottle& cmd, yarp::os::Bottle& r
                             }
                             break;
 
+                            case VOCAB_VARIABLE:
```

Comment:
I prefer if these conditions are checked inside a handleIRemoteVariable function to reduce the size of the big switch case; this is how it was done for new interfaces (i.e. torque control mode etc)